app/src/main/res/layout/fragment_start.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/startFragment"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".presentation.ui.start.StartFragment">
+
+    <com.space.quizapp.presentation.ui.start.custom_view.StartVectorView
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dimen_460dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:gravity="center_horizontal"
+        android:orientation="vertical">
+
+        <androidx.appcompat.widget.AppCompatTextView
+            android:id="@+id/textView"
+            style="@style/message_welcome_text_style"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dimen_132dp"
+            android:layout_marginBottom="48dp"
```

Comment:
გამოიყენე constraint chains მარჯინების მაგივრად

Reply:
done in this pr: https://github.com/iTarley/QuizApp-SpaceIntern/pull/6